src/core/src/service_interfaces/LifecycleManager.py

```diff
@@ -91,7 +91,7 @@ def update_core_sequence(self, completed=False):
                          'action': self.execution_config.operation,
                          'completed': str(completed),
                          'lastHeartbeat': str(self.env_layer.datetime.timestamp()),
-                         'processIds': [os.getpid()]}
+                         'processIds': [os.getpid()] if not completed else []}
```

Comment:
Since we always check the value for 'completed' first and fetch processIds only when completed is set to 'False', do we need to reset processIds for a completed = 'True' process?

Reply:
This is just defensive coding and trying to stick to attention to detail in artifacts generated.